packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerBody.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import { neutral } from '@guardian/src-foundations';
+import { from } from '@guardian/src-foundations/mq';
+import { Hide } from '@guardian/src-layout';
+import { body } from '@guardian/src-foundations/typography';
+
+import { BannerTextStyles, createBannerBodyCopy } from '../../common/BannerText';
+
+// ---- Component ---- //
+
+interface MomentTemplateBannerBodyProps {
+    messageText: (JSX.Element | JSX.Element[])[];
```

Comment:
The RRCP has moved away from using `messageText` in favour of the new `paragraphs` array attribute. At some point we'll be removing `messageText` entirely from RRCP, SDC and all the test/variant records (as/when we shift them from S3 JSON to a new database).

Reply:
oh good to know - I'll update to paragraphs 👍